docs/java/features/odata/use-vdm.md

```diff
@@ -101,6 +101,25 @@ Request parameters:
    - `ignoreAnyVersionIdentifier()` will instead always send a `*` which acts as a wildcard to match all ETags.
 - All operations allow for adding custom headers via `withheader(...)`
 
+Navigable entities:
+- By using the `forEntity` method on service classes, you can recursively traverse navigation properties in entity sets. It's a generic API extension with type-safe methods to allow for requests along chained entities, according to their navigation properties.
+- Consider the following example:
+  ```java
+  // Create a new Trip in Vacation (id 2020) of Person (username "John")
+  // POST /ODataService/People('John')/Vacations(2020)/Trips
```

Comment:
For consistency I would suggest to separate the Java code and the Http request it produces.

Reply:
The HTTP request is not mentioned anywhere else. There is no "consistency" (?)

Reply:
Yes, I meant rather the URI as in [the example for select](https://sap.github.io/cloud-sdk/docs/java/features/odata/use-generated-odata-vdm-v2-v4#select).

Reply:
Opposing to the "example for select" this example focuses on URI path and not URI query.
Maybe my updated content better fits the page.